### DISCUSSION TITLE
Will You add configuration to new model SMG II 6.2kw?

### DISCUSSION THREAD

**Discussion by studiowena** [syssi/esphome-smg-ii#51] [Ideas]
Hi,

Will You add configuration to new model of SMG 6.2kw?

**Comment by syssi** [comment]
Please ask the seller or manufacturer about a protocol description. As soon as some details are available about the supported registers a example configuration could be added/created easily.

**Reply by studiowena** [reply]
i'll ask, for now i have only manual pdf

**Reply by studiowena** [reply]
they told me tomorrow will send to email :) we will see

**Comment by studiowena** [comment]
Hi,

They give me manual + rs485 protocol but i think its only for BMS. Please look at this.

[RS485-protocol-pylon-low-voltage(3.5)(1).pdf](https://github.com/syssi/esphome-smg-ii/files/13818645/RS485-protocol-pylon-low-voltage.3.5.1.pdf)
[SMG-II-6.2KP User Manual.pdf](https://github.com/syssi/esphome-smg-ii/files/13818646/SMG-II-6.2KP.User.Manual.pdf)

Could u tell me what i have to ask them for? Maybe they dont understand :)

**Comment by studiowena** [comment]
ok i got rs232 description and is dated 2021 so i think its same as SMG II 5.6kw, please look at this

[SMG-RS232协议 Communication Protocol V1.0.pdf](https://github.com/syssi/esphome-smg-ii/files/13826945/SMG-RS232.Communication.Protocol.V1.0.pdf)

**Comment by studiowena** [comment]
Have you had time to look at the inverter instructions?

**Reply by syssi** [reply]
IMO the project is compatible with your inverter model because the registers are the same.

**Reply by studiowena** [reply]
ok im waiting for delivery and i'll let You know

**Comment by Pajaaaa** [comment]
hello, I have the 6.2kw version, but I have a problem with communication, I can't get anything from the converter, see photo for connection. Has anyone managed to communicate with this version of the converter via esp?
![IMG_4895](https://github.com/syssi/esphome-smg-ii/assets/114132701/1fba464e-4084-4de9-9f3e-58425fc65494)
![IMG_4906](https://github.com/syssi/esphome-smg-ii/assets/114132701/13d48a99-9414-4eea-a2e9-04985ebb5820)
![as](https://github.com/syssi/esphome-smg-ii/assets/114132701/9c1c80e9-71c1-4716-be2b-59a9f2edf1b1)

**Reply by Pajaaaa** [reply]
Syssi ?

**Reply by MBRetrofit** [reply]
Rx/Tx are on the wrong pins on your ESP board
